src/sql/platform/accounts/common/accountActions.ts

```diff
@@ -150,3 +151,31 @@ export class RefreshAccountAction extends Action {
 		}
 	}
 }
+
+/**
+ * Action to sign out of GitHub Copilot account
+ */
+export class GitHubCopilotSignOutAction extends Action {
+	public static ID = 'account.github.copilot.sign.out';
+	public static LABEL = localize('githubCopilotSignOut', "Sign out of GitHub Copilot");
+	public account?: azdata.Account;
```

Comment:
This doesn't seem to be used anymore. If it is make sure it's being set correctly - and that the type matches what the value can actually be.